src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -524,6 +516,18 @@ private void AddFunctionDescriptors()
             {
                 _logger.AddingDescriptorProviderForLanguage(_workerRuntime);
                 _descriptorProviders.Add(new RpcFunctionDescriptorProvider(this, _workerRuntime, ScriptOptions, _bindingProviders, _functionDispatcher, _loggerFactory, _applicationLifetime));
+                AddCodelessDescriptor();
+            }
+        }
+
+        /// <summary>
+        /// Adds a DotNetFunctionDescriptorProvider to the list of descriptors if any function metadata has language set to "codeless" in it.
```

Comment:
Can you please explain a little more about the specific purpose of codeless? invokes and descirptors are specific to  DotNet? Will this ever extend to non-dot languages?

Reply:
Discussed offline.